docs/reference/indices/shrink-index.asciidoc

```diff
@@ -125,9 +134,11 @@ NOTE: By default, with the exception of `index.analysis`, `index.similarity`,
 and `index.sort` settings, index settings on the source index are not copied
 during a shrink operation. With the exception of non-copyable settings, settings
 from the source index can be copied to the target index by adding the URL
-parameter `copy_settings=true` to the request.
+parameter `copy_settings=true` to the request. Note that `copy_settings` can not
```

Comment:
I'm confused @talevy? I thought that "can not" and "cannot" can be used interchangeably?

Reply:
I guess it is up to the reader. I just find it ambiguous since it takes on a different meaning when it proceeds expressions that have "not" in them.

>  It can not only be used this way, but another way as well.

If you see both interchangeable in that example, then I am probably wrong

Reply:
I agree with @jasontedor that `cannot` and `can not` can be used interchangably